.azuredevops/tag-builds/lapack.yml

```diff
@@ -2,28 +2,15 @@ variables:
 - group: common
 - template: /.azuredevops/variables-global.yml
 
```

Comment:
You could add the lapackVersion parameter here and feed it as a parameter to the call on line 16. Then, you're able to specify the lapackVersion when manually kicking off a build in Azure Pipelines without having to make any code changes.